extension/background.js

```diff
@@ -492,9 +492,35 @@ function loadSurvey(element, decision, timePromptShown, timePromptClicked) {
   });
 }
 
-// Trigger the new survey prompt when the participant makes a decision about an
-// experience sampling element.
+/**
+ * Record basic information about the event.
+ * @param {object} element The browser element of interest.
```

Comment:
It may not be a DOM element, it's something like an interstitial or a download tray warning etc.